client/main.go

```diff
@@ -73,11 +73,10 @@ func (f Fuzzer) run() error {
 }
 
 func (f AFLFuzzCommand) cmd() *exec.Cmd {
-	c := exec.Command(aflPath(),
+	fullCommandArgs := append([]string{
 		"-o", "output",
-		"-i", "input",
-		"-S",
-		"--", "./target")
+		"-i", "input"}, f.targetCommand...)
```

Comment:
`-S` was giving me some trouble, I'll make a task to come back and fix this stuff properly.